src/components/ProofCard.vue

```diff
@@ -63,10 +67,10 @@ export default {
         this.$emit('proofSelected', this.proof)
       }
     },
-    getProofUrl(proof) {
+    getProofUrl(proof, showThumbImage) {
       // return 'https://prices.openfoodfacts.org/img/0002/qU59gK8PQw.400.webp'  // PRICE_TAG
       // return 'https://prices.openfoodfacts.net/img/0001/lZGFga9ZOT.400.webp'  // RECEIPT
-      if (proof.image_thumb_path) {
+      if (proof.image_thumb_path && showThumbImage) {
```

Comment:
I think to simplify i'd rather have the rule only set here (and not have the 2nd param in getProofUrl)
- avoids using the prop in the template
- avoid confusion between the prop name and the method param name

```suggestion
      if (proof.image_thumb_path && this.showThumbImage) {
```

Reply:
and when you think about it, the `getProofUrl` method could also very well work without any param at all (and call this.proof instead !)

Reply:
I did actually get confused by that. So I opted to remove all params from `getProofUrl`

Thanks for the suggestion

Reply:
also did a last small fix and moved getProofUrl to computed (instead of methods)